DB/src/main/java/io/deephaven/db/util/AbstractScriptSession.java

```diff
@@ -204,4 +205,15 @@ public TableDefinition getTableDefinition(final String var) {
     public VariableProvider getVariableProvider() {
         return this;
     }
+
+
+    @Override
+    public final void setVariable(String name, Object value) {
```

Comment:
This means that anything that is deliberately put in the `QueryScope` (rather than via the binding / Python globals) will be managed.

Reply:
putting something deliberately in the query scope makes it accessible to scripts … and thus I think needs to be pinned --- that was a conclusion we agree on, right?

Reply:
I thought we concluded yesterday that we didn't want to interfere with existing mechanisms used by script writers, and instead make the mechanism used by `BindTableToVariable` special. That is, two method calls:
```
setVariable
livenessScope.manage
```